### PR TITLE
Service name as var.

### PR DESCRIPTION
**Pull request by SpacePossum**
This allows for registering the service to the application with a name of the users' liking.
Ping @tiemevanveen 

closes https://github.com/GeckoPackages/GeckoSilexMemcacheService/issues/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,21 @@ $logger = $app['memcache']->getLogger();
...
+## Custom name registering / multiple services
+
+You can register the service using a name other than the default name `memcache`.
+The same method can be used to register multiple services on your application.
+Pass the name at the constructor of the service and use the same name as prefix for the configuration.
+For example:
+
+```php
+
+$app->register(new MemcachedServiceProvider('memcached'), array('memcached.prefix' => $prefix));
```
Comment (tiemevanveen) [inline README.md]: Had to read it twice to get it. My suggestion: 

## Custom name registering / multiple services

You can register the service using a name other than the default name `memcache`.
```php
$app->register(new MemcachedServiceProvider('my.custom.name'));
```

The same method can be used to register multiple services on your application.
If you do, make sure to also use different cache prefixes for each service.
```php

$app->register(new MemcachedServiceProvider('cache1'), array('memcached.prefix' => 'cache1'));
$app->register(new MemcachedServiceProvider('cache2'), array('memcached.prefix' => 'cache2'));

// usage
$app['cache1']->set('foo', 'bar');
$app['cache2']->set('foo', 'baz');

$app['cache1']->get('foo');
$app['cache2']->get('foo');

```

**Comment by tiemevanveen** [comment]
Nice work :) 👍

File: README.md (hunk 1 above)
Comment (SpacePossum) [inline README.md]:
```php
$app->register(new MemcachedServiceProvider('cache1'), array('cache1.prefix' => 'abc'));
$app->register(new MemcachedServiceProvider('cache2'), array('cache2.prefix' => 'def'));
```

it seems the docs are indeed not clear ;)

**Comment by tiemevanveen** [comment]
see https://github.com/SpacePossum/GeckoSilexMemcacheService/pull/1

**Pull request merged**